Source/ISHRemote/Trisoft.ISHRemote/Objects/Public/IshSession.cs

```diff
@@ -80,54 +83,59 @@ public class IshSession : IDisposable
         /// Creates a session object holding contracts and proxies to the web services API. Takes care of username/password and 'Active Directory' authentication (NetworkCredential) to the Secure Token Service.
         /// </summary>
         /// <param name="webServicesBaseUrl">The url to the web service API. For example 'https://example.com/ISHWS/'</param>
-        public IshSession(ILogger logger, string webServicesBaseUrl)
```

Comment:
Why did you remove 'IshSession(ILogger logger, string webServicesBaseUrl)' overload, as it essentially breaks DLL compatibility.